sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeBatchActionsAsyncClient.java

```diff
@@ -71,6 +77,8 @@
 import static com.azure.core.util.tracing.Tracer.AZ_TRACING_NAMESPACE_KEY;
 
 class AnalyzeBatchActionsAsyncClient {
+    private static final String REGEX_ACTION_ERROR_TARGET = "#/tasks/(\\w+)/(\\d+)";
```

Comment:
Perhaps you can make this more strict by replacing the `w+` with an alternative of the actual expected words.